app/services/paperless_service.py

```
@@ -439,6 +440,123 @@ async def get_task_status(self, task_id: str) -> Dict[str, Any]:
+
+                        # Wait before next poll with exponential backoff
+                        await asyncio.sleep(poll_interval)
+                        poll_interval = min(poll_interval * 1.5, max_poll_interval)
+                        continue
+
+                    else:
+                        logger.warning(f"Unknown task status: {parsed.status}")
+                        await asyncio.sleep(poll_interval)
+                        continue
+
+            except Exception as e:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)